WooCommerce/WooCommerceTests/Model/CustomHelpCenterContentTests.swift

```diff
@@ -0,0 +1,39 @@
+import XCTest
+@testable import WordPressAuthenticator
+@testable import WooCommerce
+
+final class CustomHelpCenterContentTests: XCTestCase {
+
+    func test_step_key_has_correct_rawValue() {
+        let sut = CustomHelpCenterContent.Key.step
+        XCTAssertEqual(sut.rawValue, "source_step")
+    }
+
+    func test_flow_key_has_correct_rawValue() {
+        let sut = CustomHelpCenterContent.Key.flow
+        XCTAssertEqual(sut.rawValue, "source_flow")
+    }
+
+    func test_url_key_has_correct_rawValue() {
+        let sut = CustomHelpCenterContent.Key.url
+        XCTAssertEqual(sut.rawValue, "help_content_url")
+    }
+
+    func test_init_using_step_and_flow_returns_valid_instance_for_enter_store_address_screen() throws {
```

Comment:
nice test! how about another test case to make sure some other combination of step and flow returns `nil`?

Reply:
Makes sense. 👍 
Done in 447761ba8f7969eb5e6ae9d6094d71e1a5079cd9